app/com/linkedin/drelephant/tuning/hbt/MRConstant.java

```diff
@@ -0,0 +1,55 @@
+package com.linkedin.drelephant.tuning.hbt;
+
+final class MRConstant {
+  static final String MAPPER_TIME_HEURISTIC = "Mapper Time";
+  static final String MAPPER_SPEED_HEURISTIC = "Mapper Speed";
+  static final String MAPPER_MEMORY_HEURISTIC = "Mapper Memory";
+  static final String MAPPER_SPILL_HEURISTIC = "Mapper Spill";
+  static final String REDUCER_TIME_HEURISTIC = "Reducer Time";
+  static final String REDUCER_MEMORY_HEURISTIC = "Reducer Memory";
+
+  static final String MAPPER_MEMORY = "mapreduce.map.memory.mb";
+  static final String MAPPER_HEAP = "mapreduce.map.java.opts";
+  static final String REDUCER_MEMORY = "mapreduce.reduce.memory.mb";
+  static final String REDUCER_HEAP = "mapreduce.reduce.java.opts";
+  static final String MAP_SPLIT_SIZE = "mapreduce.input.fileinputformat.split.maxsize";
+  static final String PIG_SPLIT_SIZE = "pig.maxCombinedSplitSize";
+  static final String NUMBER_OF_REDUCER = "mapreduce.job.reduces";
+  static final String BUFFER_SIZE = "mapreduce.task.io.sort.mb";
+  static final String SPILL_PERCENTAGE = "mapreduce.map.sort.spill.percent";
+
+  static final String MAX_VIRTUAL_MEMORY = "Max Virtual Memory (MB)";
```

Comment:
Define constant in Heuristics class and reuse it here. Same comment for all Heuristic constant.